apps/dev-recruiters/src/components/modules/LoginPage/loginPage.tsx

```diff
@@ -1,27 +1,57 @@
-import React from "react";
+import React from 'react';
+import { Icons } from '@devlaunchers/components/src/assets';
 import {
   Header,
   MessageWrapper,
   Wrapper,
-  Button,
+  LoginButton,
+  CreateAccountButton,
   ButtonWrapper,
-} from "./styledLoginPage";
-import { useRouter } from "next/router";
-export default function LoginPage() {
+  Text,
+  IconWrapper,
+} from './styledLoginPage';
+import { useRouter } from 'next/router';
+import { Icon } from '@components/common/AppDropDown/StyledDropDown';
+export default function LoginPage({ closeModal }) {
   const router = useRouter();
   return (
     <Wrapper>
       <MessageWrapper>
-        <Header>Dev Launchers Volunteer Application</Header>
+        <div
+          style={{
+            width: '100%',
+            display: 'flex',
+            justifyContent: 'flex-start',
+
+            marginBottom: 12,
+          }}
+        >
+          <IconWrapper onClick={closeModal}>
+            <Icons.Chevron height={14} width={8} />
+          </IconWrapper>
+        </div>
+        <Header>Create An Account / Sign In</Header>
+        <Text>
+          Create or sign into your account to manage the roles you’re interested
+          in!
+        </Text>
         <ButtonWrapper>
-          <Button href={process.env.NEXT_PUBLIC_GOOGLE_AUTH_URL
-            +`?redirectURL=${process.env.NEXT_PUBLIC_FRONT_END_URL}/users/me`}>
-            Create an account
-          </Button>
-          <Button href={process.env.NEXT_PUBLIC_GOOGLE_AUTH_URL
-            +`?redirectURL=${process.env.NEXT_PUBLIC_FRONT_END_URL}/users/me`}>
+          <LoginButton
```

Comment:
We already have a sign in button on the menu bar. Can we point to that one?

Reply:
I discussed with @yacineKahlerras. He's working on a button component using Tailwind, but that's going to break a few things so it's not available yet.